src/js/utils/JobUtil.js

```diff
@@ -90,6 +90,9 @@ const JobUtil = {
 
     let [schedule = {}] = job.getSchedules();
     if (schedule.id != null && schedule.cron != null) {
+      if (schedule.concurrencyPolicy == null) {
+        schedule.concurrencyPolicy = 'ALLOW';
```

Comment:
Yeah, it would be nice if the backend could add the default for us. At the moment it just fails.